clients/da-vinci-client/build.gradle

```diff
@@ -34,3 +34,7 @@ dependencies {
   testImplementation project(':internal:venice-test-common')
   testImplementation libraries.kafkaClientsTest
 }
+
+ext {
+  jacocoCoverageThreshold = 0.4
```

Comment:
I guess these various thresholds are based on what each module currently achieves, right?

It'd be good for you to rebase and check those thresholds again, since I merged #155 which might result in reducing coverage in the modules that lost some tests.